packages/tools/piw-utils-internal/tsconfig.json

```diff
@@ -3,27 +3,9 @@
     "baseUrl": "./",
     "include": ["./src"],
     "compilerOptions": {
-        "target": "es2015",
-        "lib": [
-          "es2015",
-          "dom"
-        ],
         "types": ["mendix-client", "big.js", "react-native", "jest"],
-        "moduleResolution": "node",
-        "allowSyntheticDefaultImports": true,
-        "esModuleInterop": true,
-        "strict": true,
-        "noEmitOnError": true,
-        "noFallthroughCasesInSwitch": true,
         "noImplicitAny": true,
         "noImplicitReturns": true,
-        "noImplicitThis": true,
-        "noUnusedLocals": true,
-        "noUnusedParameters": true,
-        "forceConsistentCasingInFileNames": true,
-        "skipLibCheck": true,
-        "jsx": "react",
-        "jsxFactory": "createElement",
```

Comment:
Could you elaborate on these changes? They feel out of place

Reply:
Same configurations are in the base ts, so was irrelevant. I just left the non-common properties.